shared/js/background/events.es6.js

```diff
@@ -69,6 +69,9 @@ async function onInstalled (details) {
             }
         }
 
+        // remove any orphaned session rules (can happen on extension update/restart)
```

Comment:
Perhaps expand this comment, or better we could add a JSDoc comment for the function. Either way, we should explain the real idea of this somewhere.  (e.g. "We increment the rule IDs for some session rules, starting at STARTING_RULE_ID and keep a note of the next rule ID in session storage. Sometimes (e.g. on extension update) session storage can be cleared, while session rules are not. To avoid an inconsistent state, let's clear session rules on extension update too.")